code/game/machinery/cryopod.dm

```diff
@@ -472,6 +474,23 @@ var/global/list/frozen_items = list("Alpha"=list(),"Bravo"=list(),"Charlie"=list
 			//Despawning occurs when process() is called with an occupant without a client.
 			add_fingerprint(M)
 
+/obj/machinery/cryopod/verb/return_to_lobby()
+	set name = "Return to Lobby"
+	set category = "Object"
+	set src in oview(1)
+	if(occupant != usr)
+		to_chat(usr, "<span class='warning'>You must be inside the [src] to do this.</span>")
+		return
+
+	if(alert(usr, "Do you want to send yourself back to the lobby? This will be logged.","Message","Yes","No") == "Yes")
```

Comment:
Do a `== "No"` then `return` to diminish indentation. We call that an early return.

Reply:
After the alert is called the user may no longer be inside the cryo machine. That's something to consider.

Reply:
Turns out the return was redundant and not needed at all? Go figure.